implementations/rust/ockam/ockam_command/src/identity/create.rs

```diff
@@ -26,11 +28,18 @@ pub struct CreateCommand {
     /// Vault name to store the identity key
     #[arg(long, value_name = "VAULT_NAME", global = true)]
     vault: Option<String>,
+
+    #[arg(short, long)]
```

Comment:
Let's keep the argument doc string here.

Reply:
PLMK if this doc string looks ok.

Reply:
Yes, looks good!